php-frontend/src/main/java/org/sonar/php/parser/TreeFactory.java

```diff
@@ -1711,6 +1700,10 @@ public ArrayAssignmentPatternElementTree arrayAssignmentPatternElement(Optional<
     return new ArrayAssignmentPatternElementTreeImpl(variable);
   }
 
+  public ArrayAssignmentPatternElementTree arrayAssignmentPatternElement2(Optional<Tuple<ExpressionTree, InternalSyntaxToken>> key, Tree variable) {
```

Comment:
@pynicolas  didn't you already update sslr to avoid second method?

Reply:
@vilchik-elena Not when I wrote that. Fixed in d26808c. Thanks!